.githubtemplate/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+## What happened 🤔
+Describe the big picture of your changes here to communicate to team why we should accept this pull request. 
+
+
+## Insight 👀
+Describe in details how to achieve the target feature/fixes, referenced documentation is welcome as well.
+
+
+## PoW (a.k.a Proof Of Work)💪
+Show us the proof that it worked (photo, gif...etc)
+
+<img src="URL_GOES_HERE" width=200 /> 
```

Comment:
I think this is not the latest version of our `git-template`, right?
Ref: https://github.com/nimblehq/git-template/blob/main/.github/PULL_REQUEST_TEMPLATE.md